core/src/test/scala/unit/kafka/server/LogOffsetTest.scala

```diff
@@ -52,7 +52,7 @@ class LogOffsetTest extends BaseRequestTest {
     props.put("num.partitions", "20")
     props.put("log.retention.hours", "10")
     props.put("log.retention.check.interval.ms", (5 * 1000 * 60).toString)
-    props.put("log.segment.bytes", "140")
+    props.put("log.segment.bytes", "1048576")
```

Comment:
Is this still needed in this iteration of the review? Earlier we were changing tests in this file.